internal/edge/edge.go

```diff
@@ -163,6 +163,43 @@ func (manager *Manager) startEdgeBackgroundProcessOnKubernetes(runtimeCheckFrequ
 	return nil
 }
 
+func (manager *Manager) startEdgeBackgroundProcessOnNomad(runtimeCheckFrequency time.Duration) error {
```

Comment:
looks like a copy of the k8s edge process. why not merge both, and receive (engineType) argument?

Reply:
I did it on purpose as do not mix Kube and Nomad things together.